### ISSUE TITLE
Issue with gradient computation

### ISSUE THREAD

**Issue by skmanna** [VladAleshin/pytorch#1]
train.py gives the following error -
 
_one of the variables needed for gradient computation has been modified by an inplace operation: [torch.cuda.FloatTensor [512, 2]], which is output 0 of TBackward, is at version 2; expected version 1 instead. Hint: the backtrace further above shows the operation that failed to compute its gradient. The variable in question was changed in there or anywhere later. Good luck!_
